PennyPincher/PennyPincherAndroidGesturesImporter.swift

```diff
@@ -0,0 +1,200 @@
+//
+//  PennyPincherAndroidGesturesImporter.swift
+//  PennyPincher
+//
+//  Created by Raf Cabezas on 1/30/17.
+//  Copyright © 2017 WillowTree, Inc. All rights reserved.
+//
+
+import Foundation
+
+// Example usage:
+//
+//    for gesture in PennyPincherAndroidGesturesImporter.translatedGestures(from: PennyPincherAndroidGesturesImporter.defaultAndroidFilePath ?? "") {
+//        if let template = PennyPincher.createTemplate(gesture.id, points: gesture.allPoints) {
+//            pennyPincherGestureRecognizer.templates.append(template)
+//        }
+//    }
+//
+
+public struct ImportedStrokes {
+    public let points: [CGPoint]
+    
+    public init(points: [CGPoint]) {
+        self.points = points
+    }
+}
+
+public struct ImportedGesture {
+    public let id: String
+    public let strokes: [ImportedStrokes]
+    
+    public init(id: String, strokes: [ImportedStrokes]) {
+        self.id = id
+        self.strokes = strokes
+    }
+    
+    public var allPoints: [CGPoint] {
+        return strokes.reduce([CGPoint]()) { points, stroke in
+            return points + stroke.points
+        }
+    }
+}
+
+public final class PennyPincherAndroidGesturesImporter {
+    
+    public static var defaultAndroidFilePath: String? {
+        guard let path = Bundle.main.path(forResource: "gestures", ofType: "") else {
+            print("File not found")
+            return nil
+        }
+        
+        return path
+    }
+    
+    public static func translatedGestures(from path: String, debug: Bool = false) -> [ImportedGesture] {
```

Comment:
`fromPath` would be a better argument label than `from` here, since at the point of use, you only see `from: someString`, and the `String` type does not explain the string's role as a path. You might misinterpret the string as the contents of the file.

In fact, it might be better to pass a `URL` instead of a string path. In the method above, you can get the URL from the bundle instead of the string path.